### PR TITLE
Add request for feedback from Jira &  fix wording on PractitionerRole request for feedback

### PR DESCRIPTION
**Pull request by dbojicic**
Addresses
Fixes https://github.com/hl7au/au-fhir-core/issues/119
Fixes https://github.com/hl7au/au-fhir-core/issues/120

Changes related to #120:
StructureDefinition-au-core-allergyintolerance.md
StructureDefinition-au-core-bloodpressure.html
StructureDefinition-au-core-bodyheight.html
StructureDefinition-au-core-bodytemp.html
StructureDefinition-au-core-heartrate.html
StructureDefinition-au-core-immunization.html
StructureDefinition-au-core-patient.html
StructureDefinition-au-core-practitioner.html
StructureDefinition-au-core-resprate.html
StructureDefinition-au-core-waistcircum.html

Changes related to #120:
StructureDefinition-au-core-practitionerrole.md - reworded request for feedback

### CODE REVIEW

File: input/intro-notes/StructureDefinition-au-core-allergyintolerance-intro.md [hunk 1]
```diff
@@ -22,3 +22,13 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
...
 - Refutation is not expected to be handled except as above - an appropriate negation code in `AllergyIntolerance.code` and `AllergyIntolerance.verificationStatus` of "confirmed" or "unconfirmed"
 - Where only substance is known (e.g. 111088007 \|Latex\|) and not a statement of allergy or intolerance (e.g. 300916003 \|Allergy to latex\|), the substance is sent in `AllergyIntolerance.code`
+
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove Must Support from AllergyIntolerance.asserter element. It's expected that many systems might choose not to use this field,  opting to derive its value from the Recorder entity. There's some uncertainty about how widespread this approach is across different systems' information recording processes.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44699">FHIR-44699</a>.</p>
+
+<div class="request-for-feedback"><p>Specific feedback is sought on the AllergyIntolerance.code refined subset binding proposal, based on the feedback received by vendors. This proposal recommends to:<ul>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-allergyintolerance-intro.md:28]: Too many words, leave a summary sentence and remove the rest.

File: input/intro-notes/StructureDefinition-au-core-bloodpressure-intro.md [hunk 2]
```diff
@@ -23,6 +23,5 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
   - `Observation.value[x]` element **SHOULD** be omitted
   - an Observation without a systolic or diastolic result value, **SHOULD** include a reason why the data is absent in `Observation.component.dataAbsentReason`
 
-
-
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove Must Support from Observation.performer element. This element is not flagged as must support in similar profiles in other implementation guides, such as US Core and IPS, and it lacks a counterpart in AUCDI R1 with no inclusion planned for the R2 roadmap. Given that the current guidance and description are quite general, it is proposed future revisions should focus on utilising feedback from distinct use cases (such as patient-reported versus provider-reported observations) to provide more specific guidance.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44588">FHIR-44588</a>.</p>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-bloodpressure-intro.md:26]: Too many words. Shorten.

File: input/intro-notes/StructureDefinition-au-core-bodyheight-intro.md [hunk 3]
```diff
@@ -20,6 +20,6 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 ### Profile specific implementation guidance
 {% include observation_vitalsigns_guidance.md -%}
 
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove Must Support from Observation.performer element. This element is not flagged as must support in similar profiles in other implementation guides, such as US Core and IPS, and it lacks a counterpart in AUCDI R1 with no inclusion planned for the R2 roadmap. Given that the current guidance and description are quite general, it is proposed future revisions should focus on utilising feedback from distinct use cases (such as patient-reported versus provider-reported observations) to provide more specific guidance.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44588">FHIR-44588</a>.</p>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-bodyheight-intro.md:23]: Too many words. Shorten.

File: input/intro-notes/StructureDefinition-au-core-bodyheight-intro.md [hunk 4]
```diff
@@ -20,6 +20,6 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
...
 
-
-
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove the required bindings on the Observation.valueQuantity.code element. Currently, this profiles mandates a binding to UCUM (Unified Code for Units of Measure) metric measurements. While the base FHIR profile also binds these elements to UCUM, it accommodates both metric and imperial measurements. Maintaining the required binding to only metric measurements could potentially complicate the integration for systems that primarily use imperial measurements. An alternative approach could be to change the binding status from required to preferred.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44787">FHIR-44787</a>.</p>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-bodyheight-intro.md:25]: Too many words. Shorten.

File: input/intro-notes/StructureDefinition-au-core-patient-intro.md [hunk 5]
```diff
@@ -48,4 +48,21 @@ By making Individual Gender Identity a Must Support data element, this profile e
   - Where the workflow does not support obtaining a gender identity value, it may be represented by sending the code "not-asked"
 - When exchanging concepts of sex or gender, refer to the guidance in the [Gender Harmony Implementation Guide](http://hl7.org/xprod/ig/uv/gender-harmony/) and [AU Base Patient](https://build.fhir.org/ig/hl7au/au-fhir-base//StructureDefinition-au-patient.html)
 
-
+<div class="request-for-feedback"><p>Specific feedback is sought on adding Must Support to specific parts of the HumanName in the AU Core Patient profile. Following the TDG agreement on making Patient.name.use a Must Support element, this proposal extends Must Support to:<ul>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-patient-intro.md:51]: Too many words, shorten. These proposals will change over time in the JIRA ticket.

File: input/intro-notes/StructureDefinition-au-core-patient-intro.md [hunk 6]
```diff
@@ -48,4 +48,21 @@ By making Individual Gender Identity a Must Support data element, this profile e
...
+  <li>Patient.name.text</li>
+  <li>Patient.name.family</li>
+  <li>Patient.name.given</li></ul>
+Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44818">FHIR-44818</a>.
+ </p>
+</div>
+
+<div class="request-for-feedback"><p>Request for feedback on introducing a specific AU Core profile for the individual-recordedSexOrGender FHIR extension to represent Sex Assigned at Birth values. This proposal aims to:<ul>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-patient-intro.md:59]: Shorten. Remove specifics.

File: input/intro-notes/StructureDefinition-au-core-patient-intro.md [hunk 7]
```diff
@@ -48,4 +48,21 @@ By making Individual Gender Identity a Must Support data element, this profile e
...
+  <li>Define the minimum expectations for a Recorded Sex or Gender FHIR extension when representing a Sex Assigned at Birth value</li>
+  <li>Enable Sex Assigned at Birth to be a Must Support element in AU Core Patient</li>
+  <li>Ensure conformance testing of Must Support Sex Assigned at Birth values while accommodating additional RSG values </li>
+  </ul>
+Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44856">FHIR-44856</a>.
+ </p>
+</div>
+
+<p class="request-for-feedback">Specific feedback is sought on adding Must Support to Sex Assigned at Birth element, subject to <a href="https://jira.hl7.org/browse/FHIR-44856">FHIR-44856</a> being accepted as persuasive, please see above. <br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44857">FHIR-44857</a>.</p>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-patient-intro.md:68]: Combine with above on supporting Sex Assigned at Birth.

File: input/intro-notes/StructureDefinition-au-core-practitioner-intro.md [hunk 8]
```diff
@@ -32,4 +32,14 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
     </p>
 </div>
 
-<p class="request-for-feedback">Specific feedback is sought on the proposal to remove the mandatory requirement to support _revinclude of Provenance.<br/>Please comment on <a href="https://jira.hl7.org/browse/FHIR-43850">Jira issue #FHIR-43850</a>.</p>
\ No newline at end of file
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove the mandatory requirement to support _revinclude of Provenance.<br/>Please comment on <a href="https://jira.hl7.org/browse/FHIR-43850">Jira issue #FHIR-43850</a>.</p>
+
+<p class="request-for-feedback">Specific feedback is sought on the inclusion of addresses and telecom in the Practitioner's profile. Often, these details are more relevant to a practitioner's role or workplace and not to their personal information. There are concerns that practitioners may prefer to keep their private contact details confidential.<br/>Please comment on <a href="https://jira.hl7.org/browse/FHIR-43860">Jira issue #FHIR-43860</a>.</p>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-practitioner-intro.md:37]: Remove 2nd & 3rd sentence.

File: input/intro-notes/StructureDefinition-au-core-practitioner-intro.md [hunk 9]
```diff
@@ -32,4 +32,14 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
...
+
+<div class="request-for-feedback"><p>Specific feedback is sought on changes to the AU Core Practitioner profile based on the TDG decision to set Must Support on specific parts of Human Name. The proposed change is to enhance Practitoner.name components, specifically:<ul>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-practitioner-intro.md:39]: Remove specifics.

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md [hunk 10]
```diff
@@ -24,7 +24,7 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 <div class="request-for-feedback">
     <p>Specific feedback is sought on following enhancements to search parameters:
     <ul>
-        <li>Support for searching for a practitioner by reference versus chained searching using practitioner.identifier.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-43839">FHIR-43839</a>.</li>
+        <li>Support for searching PractitionerRole using practitioner by reference, and if a search via a chained practitioner.identifier could be a more effective approach.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-43839">FHIR-43839</a>.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-43839">FHIR-43839</a>.</li>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md:27]: Remove double up of "Please comment" statement.

**Review by dt-r — CHANGES_REQUESTED** [review]
See comments.

File: input/intro-notes/StructureDefinition-au-core-allergyintolerance-intro.md (hunk 1 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-allergyintolerance-intro.md:28]: Done

File: input/intro-notes/StructureDefinition-au-core-bodyheight-intro.md (hunk 3 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-bodyheight-intro.md:23]: Done

File: input/intro-notes/StructureDefinition-au-core-bodyheight-intro.md (hunk 4 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-bodyheight-intro.md:25]: Done

File: input/intro-notes/StructureDefinition-au-core-patient-intro.md (hunk 6 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-patient-intro.md:59]: Done

File: input/intro-notes/StructureDefinition-au-core-patient-intro.md (hunk 7 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-patient-intro.md:68]: Done

File: input/intro-notes/StructureDefinition-au-core-practitioner-intro.md (hunk 8 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-practitioner-intro.md:37]: Done

File: input/intro-notes/StructureDefinition-au-core-practitioner-intro.md (hunk 9 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-practitioner-intro.md:39]: Done

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md (hunk 10 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md:27]: Done

File: input/intro-notes/StructureDefinition-au-core-bodyheight-intro.md [hunk 11]
```diff
@@ -18,8 +18,10 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
...
 
 ### Profile specific implementation guidance
-{% include observation_vitalsigns_guidance.md -%}
 
 
+{% include observation_vitalsigns_guidance.md -%}
 
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove Must Support from Observation.performer element across all AU Core Vital Signs and Measurements profiles.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44786">FHIR-44786</a>.</p>
 
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove the required binding on the Observation.valueQuantity.code element, to allow for the base FHIR profile binding that accommodates both metric and imperial measurements.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44787">FHIR-44787</a>.</p>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-bodyheight-intro.md:27]: Reword the first sentence to be:
Specific feedback is sought on the proposal to remove the binding to metric units and allow for metric or imperial measurements.

File: input/intro-notes/StructureDefinition-au-core-patient-intro.md [hunk 12]
```diff
@@ -48,4 +48,7 @@ By making Individual Gender Identity a Must Support data element, this profile e
   - Where the workflow does not support obtaining a gender identity value, it may be represented by sending the code "not-asked"
 - When exchanging concepts of sex or gender, refer to the guidance in the [Gender Harmony Implementation Guide](http://hl7.org/xprod/ig/uv/gender-harmony/) and [AU Base Patient](https://build.fhir.org/ig/hl7au/au-fhir-base//StructureDefinition-au-patient.html)
 
+<p class="request-for-feedback">Following the TDG agreement to set Must Support on parts of HumanName in AU Core Patient (noting the existing Must Support on Patient.name.use), specific feedback is sought on other parts of HumanName that should be supported.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44818">FHIR-44818</a>.</p>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-patient-intro.md:51]: Replace
Following the TDG agreement to set Must Support on parts of HumanName in AU Core Patient (noting the existing Must Support on Patient.name.use), specific feedback is sought on other parts of HumanName that should be supported.

To be 
Specific feedback is sought on the proposal to add Must Support to parts of Patient.name in addition to Patient.name.use.

File: input/intro-notes/StructureDefinition-au-core-patient-intro.md [hunk 13]
```diff
@@ -48,4 +48,7 @@ By making Individual Gender Identity a Must Support data element, this profile e
...
+
+<p class="request-for-feedback">Specific feedback is sought on adding Sex Assigned at Birth as a Must Support element in AU Core Patient.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44856">FHIR-44856</a> and on <a href="https://jira.hl7.org/browse/FHIR-44857">FHIR-44857</a>.</p>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-patient-intro.md:53]: Replace
Specific feedback is sought on adding Sex Assigned at Birth as a Must Support element in AU Core Patient.

With 
Specific feedback is sought on adding an AU Core Sex Assigned at Birth profile of the Person Recorded Sex Or Gender extension (<a href="https://jira.hl7.org/browse/FHIR-44856">FHIR-44856</a>) and making that extension a Must Support element in AU Core Patient (<a href="https://jira.hl7.org/browse/FHIR-44857">FHIR-44857</a>). Please comment in JIRA.

File: input/intro-notes/StructureDefinition-au-core-practitioner-intro.md [hunk 14]
```diff
@@ -32,4 +32,8 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
     </p>
 </div>
 
-<p class="request-for-feedback">Specific feedback is sought on the proposal to remove the mandatory requirement to support _revinclude of Provenance.<br/>Please comment on <a href="https://jira.hl7.org/browse/FHIR-43850">Jira issue #FHIR-43850</a>.</p>
\ No newline at end of file
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove the mandatory requirement to support _revinclude of Provenance.<br/>Please comment on <a href="https://jira.hl7.org/browse/FHIR-43850">Jira issue #FHIR-43850</a>.</p>
+
+<p class="request-for-feedback">Specific feedback is sought on the inclusion of addresses and telecom in the Practitioner's profile.<br/>Please comment on <a href="https://jira.hl7.org/browse/FHIR-43860">Jira issue #FHIR-43860</a>.</p>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-practitioner-intro.md:37]: Replace
Specific feedback is sought on the inclusion of addresses and telecom in the Practitioner's profile.

With
Specific feedback is sought on removing Must Support from Practitioner.addresses and Practitioner.telecom.

File: input/intro-notes/StructureDefinition-au-core-practitioner-intro.md [hunk 15]
```diff
@@ -32,4 +32,8 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
...
+
+<p class="request-for-feedback">Following the TDG agreement to set Must Support on parts of HumanName in AU Core Practitioner, specific feedback is sought on which parts should be supported.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44819">FHIR-44819</a>.</p>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-practitioner-intro.md:39]: Replace
Following the TDG agreement to set Must Support on parts of HumanName in AU Core Practitioner, specific feedback is sought on which parts should be supported.

With 
Specific feedback is sought on the proposal on Practitioner.name including adding Must Support to parts of Practitioner and making Practitioner.name.family mandatory.

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md [hunk 16]
```diff
@@ -24,7 +24,7 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 <div class="request-for-feedback">
     <p>Specific feedback is sought on following enhancements to search parameters:
     <ul>
-        <li>Support for searching for a practitioner by reference versus chained searching using practitioner.identifier.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-43839">FHIR-43839</a>.</li>
+        <li>Support for searching PractitionerRole using practitioner by reference, and if a search via a chained practitioner.identifier could be a more effective approach.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-43839">FHIR-43839</a>.</li>
         <li>Support for searching for an organization by reference versus chained searching using organization.identifier.<br/>Please comment on <a href="https://jira.hl7.org/browse/FHIR-43839">FHIR-43839</a>.</li>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md:28]: Not search for an organization it is searching for PractitionerRole using organization.

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md [hunk 17]
```diff
@@ -24,7 +24,7 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
...
         <li>Removal of support for searching using HPI-I identifier, as it is not supported by AU Core PractitionerRole.<br/>Please comment on <a href="https://jira.hl7.org/browse/FHIR-43839">FHIR-43839</a>.</li>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md:29]: Remove the inclusion for "Removal of support for searching using HPI-I identifier" this is already approved and should have been done as part of the approved change request to remove support for HPI-I.

**Review by dt-r — CHANGES_REQUESTED** [review]
Confirmed previous fixes. Identified a number of additional changes needed that I missed in the first review :(

File: input/intro-notes/StructureDefinition-au-core-bodyheight-intro.md (hunk 11 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-bodyheight-intro.md:27]: Done

File: input/intro-notes/StructureDefinition-au-core-practitioner-intro.md (hunk 14 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-practitioner-intro.md:37]: Done

Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-practitioner-intro.md]: Applied change, and moved to the bullet list with other remove Must Support proposals.

File: input/intro-notes/StructureDefinition-au-core-practitioner-intro.md (hunk 15 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-practitioner-intro.md]: Done

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md (hunk 16 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md]: Done

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md (hunk 17 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md]: Done

File: input/intro-notes/StructureDefinition-au-core-bloodpressure-intro.md [hunk 18]
```diff
@@ -23,6 +23,4 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
   - `Observation.value[x]` element **SHOULD** be omitted
   - an Observation without a systolic or diastolic result value, **SHOULD** include a reason why the data is absent in `Observation.component.dataAbsentReason`
 
-
-
-
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove Must Support from Observation.performer element across all AU Core Vital Signs and Measurements profiles.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44786">FHIR-44786</a>.</p>
```
[the remark this answers is not in the document]
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-bloodpressure-intro.md]: Still says "all" but that's fine.

File: input/intro-notes/StructureDefinition-au-core-bodytemp-intro.md [hunk 19]
```diff
@@ -20,6 +20,6 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 ### Profile specific implementation guidance
 {% include observation_vitalsigns_guidance.md -%}
 
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove Must Support from Observation.performer element across all AU Core Vital Signs profiles.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44786">FHIR-44786</a>.</p>
 
-
-
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove the required binding on the Observation.valueQuantity.code element, to allow for the base FHIR profile binding that accommodates both metric and imperial measurements.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44787">FHIR-44787</a>.</p>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-bodytemp-intro.md:25]: Missed this change "Specific feedback is sought on the proposal to remove the binding to metric units and allow for metric or imperial measurements."

File: input/intro-notes/StructureDefinition-au-core-bodyweight-intro.md [hunk 20]
```diff
@@ -20,6 +20,6 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 ### Profile specific implementation guidance
 {% include observation_vitalsigns_guidance.md -%}
 
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove Must Support from Observation.performer element across all AU Core Vital Signs profiles.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44786">FHIR-44786</a>.</p>
 
-
-
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove the required binding on the Observation.valueQuantity.code element, to allow for the base FHIR profile binding that accommodates both metric and imperial measurements.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44787">FHIR-44787</a>.</p>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-bodyweight-intro.md:25]: Missed this change: Specific feedback is sought on the proposal to remove the binding to metric units and allow for metric or imperial measurements.

File: input/intro-notes/StructureDefinition-au-core-waistcircum-intro.md [hunk 21]
```diff
@@ -20,6 +20,6 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 ### Profile specific implementation guidance
 {% include observation_vitalsigns_guidance.md -%}
 
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove Must Support from Observation.performer element across all AU Core Vital Signs profiles.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44786">FHIR-44786</a>.</p>
 
-
-
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove the required binding on the Observation.valueQuantity.code element, to allow for the base FHIR profile binding that accommodates both metric and imperial measurements.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44787">FHIR-44787</a>.</p>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-waistcircum-intro.md:25]: Missed this change: Specific feedback is sought on the proposal to remove the binding to metric units and allow for metric or imperial measurements.

**Review by dt-r — CHANGES_REQUESTED** [review]
One type of change missed. Confirmed other fixes.

File: input/intro-notes/StructureDefinition-au-core-bodytemp-intro.md (hunk 19 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-bodytemp-intro.md:25]: :(  Done now

File: input/intro-notes/StructureDefinition-au-core-bodyweight-intro.md (hunk 20 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-bodyweight-intro.md:25]: Done

File: input/intro-notes/StructureDefinition-au-core-waistcircum-intro.md (hunk 21 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-waistcircum-intro.md:25]: Done

**Review by dt-r — APPROVED** [review]

**Pull request merged**